force-app/main/default/classes/main.cls

```diff
@@ -0,0 +1,28 @@
+System.debug(' Fibonacci= ' +  P0.Fibonacci(5));
```

Comment:
Don't include this file. Just the ones necessary to compile your code without any errors.

Reply:
I am sorry I accidentally pushed all the files.